source/include/azure_iot_hub_client.h

```diff
@@ -165,6 +171,15 @@ typedef struct AzureIoTHubClientReceiveContext
     } _internal;
 } AzureIoTHubClientReceiveContext_t;
 
+typedef struct AzureIoTHubClientTelemetryAckContext
```

Comment:
>AzureIoTHubClientTelemetryAckContext [](http://example.com/codeflow?start=15&length=36)

Should we name it publish ack context?

Reply:
Yup that way we allow for other features should they arise.